### DISCUSSION TITLE
Adding Forward/Backward Page Navigation support to Mouse Keys

### DISCUSSION THREAD

**Discussion by ckgresla** [minbrowser/min#2031] [Ideas]
I was hoping to get forward and backward navigation support with the mouse buttons (thumb-pressable) but could not find a feature in the preferences and do not know exactly what the keypress combination would be to add it via the Keyboard Shortcuts section -- love the application and hope to see it flourish!

**Comment by flightmansam** [comment]
What OS are you using? Mouse buttons seem to work fine on macOS.

**Reply by ckgresla** [reply]
I am on Monterey, version 12.3.1 -- the mouse I am using is a Logitech MX Master 3, any info you might be able to provide would be greatly appreciated!
